### PR TITLE
St 31/feature/header ui

### PR DESCRIPTION
**Pull request by jayDesai89**
Nothing has changed in Header UI , in this branch I just removed Add button from "My day"  page header. 
Can be merged without any issues

### CODE REVIEW

File: src/app/components/header/header.component.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <ion-header>
+  <ion-title>My Day</ion-title>
```
Comment (Dhruvkumarshah) [inline src/app/components/header/header.component.html]: Why common header have title as My Day?

File: src/app/tabs/tabs.page.scss [hunk 2]
```diff
@@ -8,4 +8,19 @@
 
 .margin-extra-left {
     margin-left: 7%;
-}
\ No newline at end of file
+}
+
+
+// :host {
+//     --background: var(--ion-item-background, var(--ion-background-color, #fff));
```
Comment (Dhruvkumarshah) [inline src/app/tabs/tabs.page.scss]: Remove code instead of commenting out

Comment (jayDesai89) [reply src/app/tabs/tabs.page.scss]: I need that for reference that is why I have added that, I will need it other changes also. so please keep it.

File: src/app/components/header/header.component.html (hunk 1 above)
Comment (jayDesai89) [reply src/app/components/header/header.component.html]: Its is not a common header, It is a local header of "My Day" page

Comment (Dhruvkumarshah) [reply src/app/components/header/header.component.html]: It is used only in my day as of today that does not mean that it is my day header. This should be common header for app. So keep it generic.

**Review by Dhruvkumarshah — APPROVED** [review]

**Comment by jayDesai89** [comment]
https://slatein.atlassian.net/wiki/spaces/SLATEINDEV/pages/348717057/Child+Components

**Review by Sharmil7171 — APPROVED** [review]

**Pull request merged**